t/ui/12-needle-edit.t

```diff
@@ -406,6 +406,23 @@ subtest 'Saving needle when "taking matches" not selected' => sub {
     check_flash_for_saving_logpackages();
 };
 
+subtest 'Saving needle when only containing ocr type area' => sub {
+    $driver->get('/tests/99938/modules/logpackages/steps/1/edit');
+    $driver->execute_script('$(\'#modal-overwrite\').removeClass(\'fade\');');
+    $driver->find_element_by_id('tag_ENV-DESKTOP-kde')->click();
+    create_needle(200, 250);
+    $driver->double_click;
+    $driver->double_click;    # the match tpye change to ocr
+    $driver->find_element_by_id('save')->click();
+    wait_for_ajax();
```

Comment:
please add a message here, e.g. `wait_for_ajax(msg => 'wait for needle area created');`